deployment/united-manufacturing-hub/values.yaml

```diff
@@ -187,10 +187,10 @@ _000_commonConfig:
     topicmap:
 # Example topic map
 #      - name: HighIntegrity
-#        topic: "^ia\\..+\\..+\\..+\\.(?!processValue).+$"
+#        topic: '^ia\..+\..+\..+\.(addMaintenanceActivity)|(addOrder)|(addParentToChild)|(addProduct)|(addShift)|(count)|(deleteShiftByAssetIdAndBeginTimestamp)|(deleteShiftById)|(endOrder)|(modifyProducedPieces)|(modifyState)|(productTag)|(productTagString)|(recommendation)|(scrapCount)|(startOrder)|(state)|(uniqueProduct)|(scrapUniqueProduct).+$'
 #        bidirectional: true
 #      - name: HighThroughput
-#        topic: "^ia\\..+\\..+\\..+\\.(processValue).*$"
+#        topic: '^ia\\..+\\..+\\..+\\.(processValue).*$'
```

Comment:
NOT '^ia\\..+\\..+\\..+\\.(processValue).*$'
BETTER: '^ia\..+\..+\..+\.(processValue).*$'